### ISSUE TITLE
Chrome tab search window (control+shift+a) only works with NVDA in Browse mode

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#18429]
### Steps to reproduce:

The search tabs window in Chrome is interesting, when you open it, the focus is in an edit field where you can type the name (or part of) for an open tab/  Below this is a list of open tabs, which gets filtered as you type more.  You can down arrow through the list BUT the caret stays in the edit box so any additional letters you type are added to the search and the list adjusts accordingly.

1. Open Chrome
2. Press control+shift+a to open the search tabs window
3. Press down arrow (or tab then down arrow) to move through the items in the list.
4. Press escape to move to browse mode
5. Now down arrow through the list.

### Actual behavior:

When moving through the list in focus mode (the default), the information about each tab is not able to be read.  When using browse mode, the information is read.  This currently requires the user to manually change modes.

### Expected behavior:

Ideally, NVDA should be able to read the details of each tab while in focus mode (enabling the form to be used as intended where you could navigate the list but still type the name of the tab to narrow the list).

If that is not possible, NVDA should switch to browse mode automatically either when down arrow is pressed, or when the user pressed tab to move from the edit box to the list initially (which isn't mandatory, but could be learnt as it is relatively intuitive).

#### NVDA version:

NVDA 2025.2 Beta 1 installed

#### Windows version:

Windows 11 (64-bit) Version: 24H2, Build: 26100.4484 

#### Name and version of other software in use when reproducing the issue:

Chrome Version: 138.0.7204.97 

#### Other information about your system:

I noticed the difference in behaviour when trying with Edge (Version: 138.0.3351.65 ) which automatically changes to browse mode when you tab out of the edit and so the list is read correctly.

**Comment by Qchristensen** [comment]
Testing this again today, it is now behaving for me.
In the meantime, my build of Chrome has updated from

Version 138.0.7204.97 (Official Build) (64-bit) 
to
Version 138.0.7204.101 (Official Build) (64-bit) 

And I've gone from NVDA 2025.2 Beta 1 to Beta 2.

I did fire up a temporary copy of Beta 1 and got it reading "Blank" for each list item the first time, but then trying again it worked.  Perhaps it is intermittent?

Is anyone else able to replicate?
